java/src/apps/AppsConfigurationManager.java

```diff
@@ -0,0 +1,22 @@
+package apps;
+
+import apps.gui3.tabbedpreferences.EditConnectionPreferencesDialog;
+import apps.gui3.tabbedpreferences.TabbedPreferencesAction;
+import jmri.Application;
+import jmri.implementation.JmriConfigurationManager;
+
+import javax.swing.*;
+
+public class AppsConfigurationManager extends JmriConfigurationManager {
+
+    @Override
+    protected void displayErrorListDialog(Object list) {
+        super.displayErrorListDialog(list);
+        (new TabbedPreferencesAction()).actionPerformed();
```

Comment:
Given that 3 other classes in `jmri` also use TabbedPreferencesAction, it would be better to provide a preferences window launching action that does not rely on `apps` instead of breaking apart JmriConfigurationManager.

Reply:
Cleaning up JmriConfigurationManager needed to be done in any case.

I will look into those other uses you mention.